apps/nextjs-website/react-components/types/HeroCounter/HeroCounter.types.ts

```diff
@@ -0,0 +1,14 @@
+import { type CommonProps, type Generic } from '../common/Common.types';
+
+export interface HeroCounterProps extends CommonProps, HeroCounterTextProps {
+  readonly background?: string | Generic;
+  readonly counterNumber?: number;
```

Comment:
```suggestion
  readonly counterNumber: number;
```
It makes sense that the `HeroCounter` component requires a number to display.
You wouldn't use this component otherwise.